src/main/java/com/coveros/selenified/Capabilities.java

```diff
@@ -267,8 +267,11 @@ private Boolean runHeadless() {
      * @param extraCapabilities any additional parameters to set for selenium
      */
     public void addExtraCapabilities(DesiredCapabilities extraCapabilities) {
+        setDesiredCapabilities();
```

Comment:
why do we call this again if we already have the capabilities set from the constructor?

Reply:
Long story short, the Capabilities object is a singleton for the Selenified suite. Since we can change/set desiredCapabilities on a per class basis, we need to reset it when we add extra capabilities, otherwise previously added ones will be retained.